src/crypto/tls/handshake_client.go

```diff
@@ -835,10 +847,14 @@ func (c *Conn) verifyServerCertificate(certificates [][]byte) error {
 	}
 
```

Comment:
@Lekensteyn, dunno what happened to the other thread; but please review the following change.